src/zoning-district-class/zoning-district-class.service.spec.ts

```diff
@@ -34,4 +40,24 @@ describe("zoning district class service unit", () => {
       ).not.toThrow();
     });
   });
+
+  describe("getZoningDistrictClassesById", () => {
+    it("should return a zoning district class by id", async () => {
+      const mock = zoningDistrictClassRepositoryMock.findByIdMocks[0];
+      const zoningDistrictClassesById =
+        await zoningDistrictClassService.findZoningDistrictClassById(mock.id);
+      expect(() =>
+        getZoningDistrictClassesByIdQueryResponseSchema.parse(
+          zoningDistrictClassesById,
+        ),
+      ).not.toThrow();
+    });
+
+    it.only("service should throw 'Resource Not Found' if the id is missing", async () => {
```

Comment:
Remove before flight

```suggestion
    it ("service should throw 'Resource Not Found' if the id is missing", async () => {
```